pkg/terminal/command.go

```diff
@@ -1063,10 +1068,49 @@ func setBreakpoint(t *Term, ctx callContext, tracepoint bool, argstr string) err
 	return nil
 }
 
+func setExecutionPoint(t *Term, ctx callContext, argstr string) error {
+	args := strings.SplitN(argstr, " ", 2)
+
+	requestedEp := &api.ExecutionPoint{}
+	locspec := ""
+	if len(args) != 1 {
+		return fmt.Errorf("address required")
```

Comment:
Other commands return `errors.New("not enough arguments")` in this situation, this should to for uniformity's sake.